common/OpTestHMC.py

```diff
@@ -914,6 +911,34 @@ def run_command(self, i_cmd, timeout=60):
         :param timeout: number, time out in seconds
         '''
         return self.ssh.run_command(i_cmd, timeout)
+   
+    def is_perfcollection_enabled(self):
+        '''
+        Get pef collection allowed in hmc profile
+
+        :returns: Ture if  allow_perf_collection in hmc otherwise false
+        '''
+
+        perf_collection = self.run_command("lssyscfg -m %s -r lpar --filter lpar_names=%s -F allow_perf_collection"
+                                % (self.mg_system, self.lpar_name))
+        if  perf_collection:
+            return True
+        else:
+            return False
+    
+    def hmc_perfcollection_on_off(self, enable=True):
```

Comment:
can we rename to hmc_perfcollect_configure ?

Reply:
i feel old one looks fine